web/src/styles/scss/password-meter.scss

```diff
@@ -0,0 +1,80 @@
+@import '@cmsgov/design-system-support/src/settings/index';
+
+.password-with-meter {
+  // inputs in the design system have a max width, so cap our container at that
+  // width too, to prevent floats and 100% widths from going too big
+  max-width: $input-max-width;
+
+  .password-with-meter--show-password label.ds-c-label {
+    margin: 24px 0 0 0;
+  }
+}
+
+.strength-meter {
+  position: relative;
+  height: 3px;
+  background: #ddd;
+  margin: 7px 0;
+  border-radius: 2px;
+
+  &:before,
+  &:after {
```

Comment:
okay this bit kind of blows my mind 0_o

Reply:
Me too, I copied it from the internet.  I need to add a comment here citing where it came from.

Reply:
Added a comment with the Github repo this came from, for future reference